tests/test_convert_positional_args.py

```diff
@@ -0,0 +1,107 @@
+from typing import List
+
+import pytest
+
+from optuna._convert_positional_args import convert_positional_args
+
+
+def _sample_func(*, a: int, b: int, c: int) -> int:
+    return a + b + c
+
+
+def test_convert_positional_args_decorator() -> None:
+    previous_positional_arg_names: List[str] = []
+    decorator_converter = convert_positional_args(
+        previous_positional_arg_names=previous_positional_arg_names
+    )
+
+    decorated_func = decorator_converter(_sample_func)
+    assert decorated_func.__name__ == _sample_func.__name__
+
+
+def testconvert_positional_args_user_warning() -> None:
```

Comment:
```suggestion
def test_convert_positional_args_user_warning() -> None:
```

Reply:
Thanks! Fixed in 946fdb2 with this change, `user_waning` => `future_warning`.

Reply:
Good catch, thanks!